features/cron.feature

```diff
@@ -58,5 +62,9 @@ Feature: Check crons
     When I run `wp launchcheck cron`
     Then STDOUT should contain:
       """
+      Some jobs are registered more than 10 times, which is excessive and may indicate a problem with your code.
```

Comment:
@danielbachhuber any reason not to have the `These jobs include: ' . implode( ', ', $excessive_jobs ),` part of the message in the test?